micronaut/src/main/kotlin/de/codecentric/hikaku/converters/micronaut/MicronautConverter.kt

```diff
@@ -54,7 +54,9 @@ class MicronautConverter(private val packageName: String) : AbstractEndpointConv
                 pathParameters = extractPathParameters(path, method),
                 headerParameters = extractHeaderParameters(method),
                 consumes = extractConsumes(resource, method),
-                produces = extractProduces(resource, method)
+                produces = extractProduces(resource, method),
+                deprecated = method.isAnnotationPresent(Deprecated::class.java)
+                    || method.declaringClass.isAnnotationPresent(Deprecated::class.java)
```

Comment:
During the review I just realized that I missed to add the supported Features to the MicronautConverter. Right now only the base features are activated.

I just made a commit to fix that. Maybe you could rebase from that and add the the deprecation feature.